scripts/ms

```diff
@@ -82,28 +63,79 @@ def init(opts):
         return json.loads(s.replace("'", '"'))
 
     conf.update({"calendar": conf.get('calendar', 'standard'),
-                 "nprocs": opts.n_processes,
                  "method": method,
                  "out_dir": out_dir,
                  "out_state": out_state,
                  "state": state_file,
                  "domain": domain_file,
                  "forcing": forcing_files,
-                 "verbose": opts.verbose * logging.INFO})
+                 "verbose": verbose * logging.INFO})
     conf['out_vars'] = to_list(conf.get('out_vars', '[]'))
     conf['iter_dims'] = to_list(conf.get('iter_dims', '["lat", "lon"]'))
-    if opts.time_grouper is not None:
-        conf['time_grouper'] = opts.time_grouper
+    conf['time_grouper'] = time_grouper if time_grouper else None
     conf = {k: v for k, v in conf.items() if v != []}
     return conf
 
 
-def main():
+def setup_dask(scheduler, num_workers):
```

Comment:
@arbennett - I'd like to test this out for each scheduler with varying number of workers using the benchmarking dataset you had used previously. Can you point me to that dataset?